sorc/checkout.sh

```diff
@@ -30,6 +34,8 @@ function checkout() {
   #   topdir [default: $(pwd)]: parent directory to your checkout
   #   logdir [default: $(pwd)]: where you want logfiles written
   #   CLEAN [default: NO]:      whether to delete existing directories and create a fresh clone
+  #   CHECKOUT_GSI [default: NO]:  whether to clone GSI repository (GSI-based DA)
+  #   CHECKOUT_GDAS [default: NO]: whether to clone GDASApp repository (UFS-based DA)  
```

Comment:
This is the docblock for the function, not the whole script. `CHECKOUT_GSI` and `CHECKOUT_GDAS` are not used by the function.

Reply:
`CHECKOUT_GSI` and `CHECKOUT_GDAS` lines removed.   Done at [53befc3](https://github.com/NOAA-EMC/global-workflow/commit/53befc3d11a5b6e8ff5404e8eed2ecd48a6bc0bc)